ProjectShop/WishList/serializers.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import serializers
+
+from ProductApp.serializers import ProductSerializer
+from UserApp.models import User
+
+
+class WishListSerializer(serializers.ModelSerializer):
+    wishlist = ProductSerializer(many=True, read_only=True)
```

Comment:
can we do that serializer will give us product names?

Reply:
using many=True converts multiple objects in one list, so if we have 3 products in wishlist, the serialized object will have list of 3 fully described products

Reply:
it does not seems right for me to serialize only Product name in WishList